docs/pages/_app.js

```diff
@@ -206,17 +205,26 @@ function AppWrapper(props) {
   const pageContextValue = React.useMemo(() => {
     const { activePage, activePageParents } = findActivePage(pages, router.pathname);
     const languagePrefix = pageProps.userLanguage === 'en' ? '' : `/${pageProps.userLanguage}`;
+    const productIdSubpathMap = {
+      introduction: '/x/introduction',
+      'x-data-grid': '/x/react-data-grid',
+      'x-date-pickers': '/x/react-date-pickers',
+      'x-charts': '/x/react-charts',
+      'x-tree-view': '/x/react-tree-view',
+    };
+
+    const getVersionOptions = (id, versions) =>
+      versions.map((version) => ({
+        text: version,
+        href: `https://${version !== process.env.LIB_VERSION ? `${version}.` : ''}mui.com${languagePrefix}${productIdSubpathMap[id]}/`,
+        current: version.length > 4, // excludes 'next' and all 'vx' versions, but includes 'vx.x.x'
```

Comment:
The condition `version !== process.env.LIB_VERSION` will never be true since the one you want to validate has a `v`

Curently you get `'7.21.0' !== 'v7.21.0'` which is false

I think it would be easier if you do an early return with

```js
if( version === `v${process.env.LIB_VERSION}` ) {
  return {
    current: true,
    text: version,
    href: `${languagePrefix}${productIdSubpathMap[id]}/`,
  }
}
return {
  text: version,
  href: `https://${version}.mui.com${languagePrefix}${productIdSubpathMap[id]}/`,
}
```

I assumed that if you are on the current version, you don't need to specify `https://.../` because it can be a relative link